Yoast/wordpress-seo
# Removed deleted replace vars from help center

## Summary

This PR can be summarized in the following changelog entry:

* Removed deleted replacement variables from the help center.

## Relevant technical choices:

* N/A.

## Test instructions

This PR can be tested by following these steps:

* Visit spots where the help center shows the snippet variables tab (post page or search appearance page), and check if the named snippet variables are no longer on it.

## Quality assurance


Fixes #9893
